pkg/scheduler/core/generic_scheduler.go

```diff
@@ -64,7 +64,11 @@ func (g *genericScheduler) Schedule(ctx context.Context, placement *policyv1alph
 	}
 	klog.V(4).Infof("feasible clusters scores: %v", clustersScore)
 
-	clusters := g.selectClusters(clustersScore, placement.SpreadConstraints, feasibleClusters)
+	clusters, err := g.selectClusters(clustersScore, placement, spec)
+	if err != nil {
+		return result, fmt.Errorf("failed to select clusters: %v", err)
+	}
+	klog.V(4).Infof("selected clusters: %v", clusters)
```

Comment:
`%v` just print the address of elem in `clusters`, it would not be helpful.

Reply:
> `%v` just print the address of elem in `clusters`, it would not be helpful.

the CRD cluster has a String function:
https://github.com/karmada-io/karmada/blob/3a5d6d132a056f95fb7dad6159cd4b7abb0a5179/pkg/apis/cluster/v1alpha1/cluster_utils.go#L4-L6